src/osuTK/osuTK.NS20.csproj

```diff
@@ -1,16 +1,38 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
This should be excluded from the new csprojs.

Reply:
That was just there so Notepad++ recognizes it as an XML file (because that's essentially what it is), but sure. I'll remove that.